eng/versioning/version_client.txt

```diff
@@ -38,6 +38,7 @@ com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-anomalydetector;3.0.0-beta.5;3.0.0-beta.6
 com.azure:azure-ai-contentsafety;1.0.0-beta.1;1.0.0-beta.2
+com.azure:azure-ai-documentintelligence;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
https://github.com/Azure/azure-sdk-for-java/pull/37634

This PR should unblock the error created by azure-ai-documentintelligence

Reply:
Thanks for the notice. Now I no longer need to have the fix in this PR :-)